src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,124 @@
 package core.basesyntax;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 /**
- * Feel free to remove this class and create your own.
+ Feel free to remove this class and create your own.
  */
+
 public class HelloWorldTest {
+    private static RegistrationService registrationService;
+    private static final int THREE_ELEMENTS = 3;
+    private static final int ONE_ELEMENTS = 1;
+
+    @BeforeAll
+    static void create() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void nullList() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_ageLess18_NotOk() {
+        User userLess18 = new User();
+        userLess18.setLogin("UserLess18");
+        userLess18.setPassword("UserLess18");
+        userLess18.setAge(17);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(userLess18),
+                "Age has to be between 18 and 120");
+    }
+
+    @Test
+    void register_PasswordLess6Numbers_NotOk() {
+        User user = new User();
+        user.setLogin("User");
+        user.setPassword("User");
+        user.setAge(30);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user),
+                "Password less 6 symbols is input");
+    }
+
+    @Test
+    void register_IfExist_NotOk() {
+        User user = new User();
+        user.setLogin("User12345");
+        user.setPassword("User@12345");
+        user.setAge(25);
+        registrationService.register(user);
+        assertEquals(ONE_ELEMENTS, Storage.people.size(), "Size not equals");
+        User user2 = new User();
+        user2.setLogin("User12345");
+        user2.setPassword("User@12345");
+        user2.setAge(25);
+        assertThrows(RuntimeException.class,
+                () -> registrationService.register(user2), "Same user was putted in storage");
```

Comment:
```suggestion
        User user = new User();
        user.setLogin("User12345");
        user.setPassword("User@12345");
        user.setAge(25);
        storageDao.add(user); 
        assertThrows(RuntimeException.class,
                () -> registrationService.register(user), "Same user was putted in storage");
```
Create a StorageDao object in the class and use it to add a user to the database